### PR TITLE
Azure Orchestrator: load_environment: add error message

### PR DESCRIPTION
**Pull request by adityagesh**
Add error message if fetching vm details from vms_map fails

### CODE REVIEW

File: lisa/sut_orchestrator/azure/common.py [hunk 1]
```diff
@@ -1448,7 +1448,9 @@ def load_environment(
         assert isinstance(node, RemoteNode)
 
         vm = vms_map.get(node.name)
-        assert_that(vm).is_not_none()
+        assert_that(vm).described_as(
+            f"VM details for {node.name} is {vm}. This is an unexpected failure"
```
Comment (squirrelsc) [inline lisa/sut_orchestrator/azure/common.py]: This message just prints values, no reason or solution. It should be like, "Cannot find vm with name {node.name}. Make sure the VM exists in resource group {resource_group_name}"

Comment (adityagesh) [reply lisa/sut_orchestrator/azure/common.py]: @squirrelsc I initially thought of something like this, but 
vms = compute_client.virtual_machines.list(resource_group_name) 
this api dynamically fetches the vm from the resource group, so it is not possible to get a vm that is not available in rg, right?

Comment (squirrelsc) [reply lisa/sut_orchestrator/azure/common.py]: Yes, so if the vm is not found. The error message indicates the expected vm in a resource group.

Comment (adityagesh) [reply lisa/sut_orchestrator/azure/common.py]: Okay, Updated

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**